.travis.yml

```diff
@@ -4,10 +4,11 @@ node_js:
   - "7"
   - "8"
 before_install:
-  - wget http://git.savannah.gnu.org/cgit/parallel.git/plain/src/parallel # Minimal installation
-  - chmod 755 parallel
+  - # Minimal installation as desdcribed in https://github.com/martinda/gnu-parallel
+  - wget http://git.savannah.gnu.org/cgit/parallel.git/plain/src/parallel
+  - chmod 700 parallel # Need rwx permissoins for user.
```

Comment:
Not sure why you need write and read permissions. `chmod + x parallel` may be sufficient.